users/kuatsure/kuatsure.c

```diff
@@ -1,35 +1,64 @@
 #include "kuatsure.h"
-
-void tmux_prefix(void) {
-  register_code(KC_LCTL);
-  register_code(KC_SPC);
-
-  unregister_code(KC_LCTL);
-  unregister_code(KC_SPC);
+#include "quantum.h"
+#include "action.h"
+#include "version.h"
+
+qk_tap_dance_action_t tap_dance_actions[] = {
+  [TD_LBRC] = ACTION_TAP_DANCE_DOUBLE(KC_LBRC, KC_LT),
+  [TD_RBRC] = ACTION_TAP_DANCE_DOUBLE(KC_RBRC, KC_GT)
+};
+
+__attribute__ ((weak))
+bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
+  return true;
 }
 
-void tmux_pane_zoom(void) {
-  tmux_prefix();
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case KB_MAKE:
+      if (!record->event.pressed) {
+        SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP SS_TAP(X_ENTER));
+      }
+      return false;
+      break;
+
+    case KB_VRSN:
+      if (!record->event.pressed) {
+        SEND_STRING(QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION);
+      }
+      return false;
+      break;
+
+    case KB_FLSH:
+      if (!record->event.pressed) {
+        SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP
+          #if  (defined(BOOTLOADER_DFU) || defined(BOOTLOADER_LUFA_DFU) || defined(BOOTLOADER_QMK_DFU))
+            ":dfu "
+          #elif defined(BOOTLOADER_HALFKAY)
+            ":teensy "
+          #elif defined(BOOTLOADER_CATERINA)
+            ":avrdude "
+          #endif
+          SS_TAP(X_ENTER)
+        );
+
+        reset_keyboard();
+      }
+      return false;
+      break;
+  }
 
-  register_code(KC_Z);
-  unregister_code(KC_Z);
+  return process_record_keymap(keycode, record);
 }
 
-void tmux_pane_switch(uint16_t keycode) {
-  tmux_prefix();
-
-  register_code(KC_Q);
-  unregister_code(KC_Q);
-
-  register_code(keycode);
-  unregister_code(keycode);
+void tmux_prefix(void) {
+  SEND_STRING(SS_LCTRL(" "));
```

Comment:
Alternatively, you could also do: 
```suggestion
  tap_code16(LCTL(KC_SPC));
```